frontend/src/pages/RunDetailsRouter.tsx

```diff
@@ -40,7 +40,7 @@ export default function RunDetailsRouter(props: RunDetailsProps) {
     pipelineManifest = JsYaml.safeDump(v2Run.pipeline_spec);
   }
 
-  const pipelineVersionId = v2Run?.pipeline_version_id;
+  const pipelineVersionId = v2Run?.pipeline_version_reference?.pipeline_version_id;
```

Comment:
Do we need to change https://github.com/kubeflow/pipelines/blob/d53307551ffe133895f9475419dd436755cffa31/frontend/src/pages/RunDetailsRouter.tsx#L52 to provide pipelineId as well?

Reply:
Yes, this changes is needed and it will be included in the pipeline API integration.